types/v56/types.go

```diff
@@ -473,23 +473,23 @@ type AdminVdc struct {
 // Description: Represents a Provider VDC.
 // Since: 0.9
 type ProviderVdc struct {
-	Xmlns        string `xml:"xmlns,attr"`
-	HREF         string `xml:"href,attr,omitempty"`
-	Type         string `xml:"type,attr,omitempty"`
-	ID           string `xml:"id,attr,omitempty"`
-	OperationKey string `xml:"operationKey,attr,omitempty"`
-	Name         string `xml:"name,attr"`
-	Status       int    `xml:"status,attr,omitempty"` // -1 (creation failed), 0 (not ready), 1 (ready), 2 (unknown), 3 (unrecognized)
-
-	AvailableNetworks     *AvailableNetworks       `xml:"AvailableNetworks,omitempty"`     // Read-only list of available networks.
-	Capabilities          *Capabilities            `xml:"Capabilities,omitempty"`          // Read-only list of virtual hardware versions supported by this Provider VDC.
-	ComputeCapacity       *RootComputeCapacity     `xml:"ComputeCapacity,omitempty"`       // Read-only indicator of CPU and memory capacity.
-	Description           string                   `xml:"Description,omitempty"`           // Optional description.
-	IsEnabled             *bool                    `xml:"IsEnabled,omitempty"`             // True if this Provider VDC is enabled and can provide resources to organization VDCs. A Provider VDC is always enabled on creation.
-	Link                  *Link                    `xml:"Link,omitempty"`                  // A reference to an entity or operation associated with this object.
-	NetworkPoolReferences *NetworkPoolReferences   `xml:"NetworkPoolReferences,omitempty"` // Read-only list of network pools used by this Provider VDC.
-	StorageProfiles       *ProviderStorageProfiles `xml:"StorageProfiles,omitempty"`       // Container for references to vSphere storage profiles available to this Provider VDC.
-	Tasks                 *TasksInProgress         `xml:"Tasks,omitempty"`                 // A list of queued, running, or recently completed tasks associated with this entity.
+	//Xmlns        string `xml:"xmlns,attr"`
```

Comment:
Is this commented out with a purpose?

Reply:
I can remove it. It was there before we started using the data structure for creation/update. Such field is needed for XML, but since we are using JSON for creation/Update, it should not be there.